client/views/app/sideNav/directMessagesFlex.html

```diff
@@ -14,9 +14,9 @@
 			<div class="input-line">
 				<label for="who">{{_ "With_whom" }}</label>
 				{{#if relabelRoom}}
-					<input id="who" type="text" readonly value="{{selectedUser}}">
```

Comment:
agreed, but i'm following what RocketChat did on the privateGroupsFlex page to be consistent.  They meat selected user's name vs selected username.  I thought about changing it, but it would be harder to merge with upstream later.  I'll add a note to make it clear.